nautobot/dcim/views.py

```diff
@@ -136,6 +136,11 @@ def post(self, request):
         )
 
 
+class BaseDeviceComponentsBulkRenameView(generic.BulkRenameView):
+    def get_selected_objects_parents_name(self, selected_objects):
+        return selected_objects.first().parent.name
```

Comment:
Should this be `.device `rather than `.parent `since InventoryItem does not have `parent @property `defined, so probably safer to just do `.device` or implement `parent @property` on InventoryItem?

Reply:
Ugh, good catch. In `next` `InventoryItem.parent` isn't the device, it's the parent InventoryItem if any. There's probably a bug [here](https://github.com/nautobot/nautobot/blob/next/nautobot/core/views/generic.py#L1491) as a result of that. You're right, we'll need to explicitly go to `.device` or `.virtual_machine` instead of relying on `.parent`.